tap_bing_ads/__init__.py

```diff
@@ -54,6 +58,39 @@
 
 ARRAY_TYPE_REGEX = r'ArrayOf([A-Za-z0-9]+)'
 
+def should_retry_httperror(exception):
+    """ Return true if exception is required to retry otherwise return false """
+    try:
+        if isinstance(exception, ConnectionError) or isinstance(exception, ssl.SSLError):
+            return True
+        elif isinstance(exception, suds.transport.TransportError) or isinstance(exception, socket.timeout):
+            return True
+        elif type(exception) == URLError:
+            return True
+        elif type(exception) == Exception and exception.args[0][0] == 408:
+            return True
+        elif exception.code == 408:
```

Comment:
Please merge this one and above condition with or

Reply:
Merged it.